libs/auth/src/lib/components/register/register.component.ts

```diff
@@ -36,6 +41,46 @@ export class RegisterComponent {
         }
     }
 
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    onKey(event: any, type: string) {
+     if (type === 'password') {
+            this.password = event.target.value;
```

Comment:
Das ist nicht nötig, da das Input Feld mit [(ngModel)]="password" gekennzeichnet ist, und somit immer der aktuelle Wert des Input Feldes in die variable password geschrieben wird (Angular Standardverhalten)